### PR TITLE
Added the Python Example for the Delete Results API

### PR DESCRIPTION
**Pull request by Madhan-Reddy-ni**
**Justification**
In order to represent how to use the APIs created in the TestMonitor service in the SystemLink Enterprise version, there is a need to create examples that will show how to call the APIs, and what details are sent in the API request body.

**Implementation**
In this example, I have demonstrated how to use the Delete Result API and Delete Results API in Python

**Testing**
Manual testing completed

### CODE REVIEW

File: Python Examples/TestMonitor/test_data_manager_client.py [hunk 1]
```diff
@@ -67,6 +69,20 @@ def update_test_results_request(results: dict, determine_status_from_steps: bool
         "determineStatusFromSteps": determine_status_from_steps
     }
 
+def delete_results_request(result_ids, delete_steps):
+    """
+    creates a delete test results request object
+    dictionary required for deleting the existing test results.
```
Comment (mohan-ni) [inline Python Examples/TestMonitor/test_data_manager_client.py]:
```suggestion
```

File: Python Examples/TestMonitor/DeleteResults/delete_results.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+parent = os.path.dirname(current)
+sys.path.append(parent)
+
+import test_data_manager_client
+
+def main():
+
+    try:
+
+        test_result = {
```
Comment (mohan-ni) [inline Python Examples/TestMonitor/DeleteResults/delete_results.py]: Can move this to separate method

File: Python Examples/TestMonitor/DeleteResults/delete_results.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        input()
+
+        # delete test result
+        test_data_manager_client.delete_result(test_result['id'], True)
+        print(f"\nThe test result with Id = {test_result['id']} has been deleted")
+        
+        # create multiple test results
+        print("\nCreating multiple test results.\nResult Ids has been listed down below")
+        result_ids = []
+        for i in range(0,5):
```
Comment (mohan-ni) [inline Python Examples/TestMonitor/DeleteResults/delete_results.py]: Move this to separate method

File: Python Examples/TestMonitor/DeleteResults/ReadMe.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+Test Monitor Results Example
+=================
+
+This is an python example demonstrating how to use the
+SystemLink Test Monitor API to create test results and steps.
```
Comment (mohan-ni) [inline Python Examples/TestMonitor/DeleteResults/ReadMe.md]:
```suggestion
SystemLink Enterprise TestMonitor APIs to delete test results.
```

**Review by mohan-ni — CHANGES_REQUESTED** [review]

File: Python Examples/TestMonitor/test_data_manager_client.py (hunk 1 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/test_data_manager_client.py]: applied

File: Python Examples/TestMonitor/DeleteResults/delete_results.py (hunk 3 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/delete_results.py]: Moved into seperate method

File: Python Examples/TestMonitor/DeleteResults/delete_results.py (hunk 2 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/delete_results.py]: moved to separate method

File: Python Examples/TestMonitor/DeleteResults/ReadMe.md (hunk 4 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/ReadMe.md]: applied the suggestion

File: Python Examples/TestMonitor/DeleteResults/delete_results.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+parent = os.path.dirname(current)
+sys.path.append(parent)
+
+import test_data_manager_client
+
+def get_test_result():
+    test_result = {
+        "programName": "Power Test",
+        "status": {
+            "statusType": "RUNNING",
```
Comment (mohan-ni) [inline Python Examples/TestMonitor/DeleteResults/delete_results.py:27]: I think creating results with `status` as `PASSED` and `totalTimeInSeconds` as some value will be more realistic.

**Review by mohan-ni — APPROVED** [review]

File: Python Examples/TestMonitor/DeleteResults/delete_results.py (hunk 5 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/delete_results.py:27]: Updated the status as passed and the total time to 13.6 seconds

File: Python Examples/TestMonitor/DeleteResults/ReadMe.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+Running the Example
+-------------------
+
+1. Clone _or_ download and extract the [repository source](https://github.com/ni/systemlink-enterprise-examples/archive/master.zip).
+2. Install the [Python SDK](https://www.python.org/downloads/).
+
+To run the example, use the following command:
+
+```
+python <filename.py> <url> <api_key>
```
Comment (ramganesh-ni) [inline Python Examples/TestMonitor/DeleteResults/ReadMe.md]: Can we make this consistent with C# examples ?

```suggestion
python <filename.py> <server_url> <api_key>
```

File: Python Examples/TestMonitor/DeleteResults/delete_results.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
+import uuid
```
Comment (ramganesh-ni) [inline Python Examples/TestMonitor/DeleteResults/delete_results.py:10]: Please add documentation similar to what we had in create results and steps example

File: Python Examples/TestMonitor/DeleteResults/delete_results.py [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+    try:
+
+        test_result = get_test_result()
+
+        # create test result
+        response = test_data_manager_client.create_results(results=[test_result])
+        test_result = response["results"][0]
+        
+        print(f"The test result has been under part number={test_result['partNumber']} with Id = {test_result['id']}")
```
Comment (ramganesh-ni) [inline Python Examples/TestMonitor/DeleteResults/delete_results.py]:
```suggestion
        print(f"The test result has been created under part number={test_result['partNumber']} with Id = {test_result['id']}")
```

File: Python Examples/TestMonitor/DeleteResults/delete_results.py [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+        print("Press enter to delete the result")
+        input()
+
+        # delete test result
+        test_data_manager_client.delete_result(test_result['id'], True)
+        print(f"\nThe test result with Id = {test_result['id']} has been deleted")
+        
+        # create multiple test results
+        print("\nCreating multiple test results.\nResult Ids has been listed down below")
```
Comment (ramganesh-ni) [inline Python Examples/TestMonitor/DeleteResults/delete_results.py]:
```suggestion
        print("\nCreating multiple test results.\nResult Ids are listed down below")
```

File: Python Examples/TestMonitor/DeleteResults/delete_results.py [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+        result_ids = []
+        
+        result_ids = create_multiple_results(result_ids, test_result)
+        
+        print("\nThe multiple results has been created successfully")
```
Comment (ramganesh-ni) [inline Python Examples/TestMonitor/DeleteResults/delete_results.py]:
```suggestion
        print("\nMultiple test results have been created successfully")
```

**Review by ramganesh-ni — APPROVED** [review]

File: Python Examples/TestMonitor/DeleteResults/delete_results.py (hunk 7 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/delete_results.py:10]: Added the documentation

File: Python Examples/TestMonitor/DeleteResults/ReadMe.md (hunk 6 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/ReadMe.md]: applied the suggestion

File: Python Examples/TestMonitor/DeleteResults/delete_results.py (hunk 8 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/delete_results.py]: applied the suggestion

File: Python Examples/TestMonitor/DeleteResults/delete_results.py (hunk 9 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/delete_results.py]: applied the suggestion

File: Python Examples/TestMonitor/DeleteResults/delete_results.py (hunk 10 above)
Comment (Madhan-Reddy-ni) [reply Python Examples/TestMonitor/DeleteResults/delete_results.py]: applied the suggestion

**Pull request closed without merging**